src/app/components/impersonification-notification/impersonification-notification.coffee

```diff
@@ -0,0 +1,49 @@
+ImpersonificationNotificationCtrl = ($scope, $log, toastr, MnoeUserAccessRequests) ->
+  'ngInject'
+
+  $scope.isLoading = true
+
+  $scope.initialize = (requests) ->
+    $scope.requests = requests
+    $scope.isLoading = false
+
+  $scope.requesterName = (request) ->
+    request.requester.name + ' ' + request.requester.surname
+
+  $scope.denyAccess = (index, request) ->
```

Comment:
`request` is the functional object here, the arguments order should be switched.